packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -233,16 +253,21 @@ class AnimationController extends Animation<double>
   }
 
   /// Stops running this animation.
+  ///
+  /// This does not trigger any notifications. The animation stops in its
+  /// current state.
   void stop() {
     _simulation = null;
     _lastElapsedDuration = null;
     _ticker.stop();
   }
 
-  /// Stops running this animation.
+  /// Release the resources used by this object. The object is no longer usable
+  /// after this method is called.
   @override
   void dispose() {
-    stop();
+    _ticker.dispose();
+    super.dispose();
```

Comment:
Does the base class not have `@mustCallSuper`?  If not, should we introduce it here?